modules/olm-installing-opm.adoc

```diff
@@ -58,7 +58,7 @@ endif::[]
 $ chmod +x ./opm
 ----
 
-. Place the file anywhere in your `PATH`, such as `/usr/local/bin/`.
+. Place the file anywhere in your `PATH`, such as `/usr/local/bin/`:
```

Comment:
this doc should probably be updated to include information about windows and mac versions (same process but the oc command will be `--path /usr/bin/darwin-amd64-opm:.` or `--path /usr/bin/windows-amd64-opm:.`

Reply:
Broke this off into a separate PR for review: https://github.com/openshift/openshift-docs/pull/26314